elastic/beats
# Fix used memory calculations in docker/memory

## What does this PR do?

This brings the memory usage reporting for `docker/memory` in line with `docker stat`, as well as `cadvisor` and `containerd` by calculating used memory as `used - inactive_file`. I'm marking this a bug, since we were the only ones using this more simplistic usage calculation.

## Why is it important?

This brings us in line with how container memory usage is calculated elsewhere.

## Checklist


## How to test this PR locally

- Pull down and build
- Run on a system with docker and containers running
- Make sure the value for container memory `usage` in `docker/memory` is a reasonable value.

## Related issues

- closes https://github.com/elastic/beats/issues/26751
